bin/datagen.py

```diff
@@ -26,8 +36,15 @@
     if("ForcedSource" in spec):
         row_counts["ForcedSource"] = None
 
-    tables = dataGen.make_chunk(chunk_id, num_rows=row_counts)
+    seed = 1
+    edge_width = 0.017 # degrees
+    tables = dataGen.make_chunk(chunk_id, num_rows=row_counts, seed=seed,
+                                edge_width=edge_width, edge_only=edge_only)
+
+    print("tables=", tables)
 
     for table_name, table in tables.items():
-        table.to_parquet("chunk{:d}_{:s}.parquet".format(chunk_id, table_name))
+        edgeType = "CT"  # complete
+        if edge_only: edgeType = "EO" # edge only
+        table.to_parquet("chunk{:d}_{:s}_{:s}.parquet".format(chunk_id, edgeType, table_name))
```

Comment:
If you change this to `.to_csv()`, then there's no need to invoke a separate conversion step. I tried this for an example chunk and the results look the same to me (specifying `header=False` and `index=False`).

Reply:
Concur here.  The csv support in pandas, with nullable integer dtypes, makes it quite pleasant.